src/components/idleMembers/card/index.tsx

```diff
@@ -1,17 +1,22 @@
-import { FC } from 'react';
+import { FC, SyntheticEvent } from 'react';
 import classNames from '@/components/idleMembers/card/card.module.scss';
+import { DUMMY_PROFILE } from '@/components/constants/display-sections.js';
+
+const imgUrl = process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL;
```

Comment:
Now that you know the convention for constant values being uppercase, do you feel it should be used here as well? 😉

Reply:
I have to put it in constants folder and uppercase imgUrl , I guess😅